images/win/Windows2022-Readme.md

```diff
@@ -92,25 +93,20 @@
 - Hub CLI 2.14.2
 
 ### Rust Tools
-- Cargo 1.57.0
-- Rust 1.57.0
-- Rustdoc 1.57.0
+- Cargo 
+- Rust 
+- Rustdoc 
 - Rustup 1.24.3
 
 #### Packages
 - bindgen 0.59.2
-- cargo-audit 0.16.0
-- cargo-outdated v0.10.2
-- cbindgen 0.20.0
-- Clippy 0.1.57
-- Rustfmt 1.4.37
```

Comment:
@V-Zabayrachny need to understand where is the packages

Reply:
@miketimofeev There is an issue with the version retrieval, this will be fixed as part of this PR: https://github.com/actions/virtual-environments/pull/4774/files